uxarray/core/dataarray.py

```diff
@@ -262,6 +265,36 @@ def nearest_neighbor_remap(
 
         return _nearest_neighbor_uxda(self, destination_obj, remap_to, coord_type)
 
+    def inverse_distance_weighted_remap(
+        self,
+        destination_obj: Union[Grid, UxDataArray, UxDataset],
+        remap_to: str = "nodes",
+        coord_type: str = "spherical",
+        power=2,
+        k_neighbors=8,
```

Comment:
```suggestion
        k=8,
```

Can we change this throughout to be consistent with the `k` used in our `.query`.